assets/css/ecommerce.css

```diff
@@ -37,13 +37,13 @@ body.woocommerce_page_wc-bridge-landing-page #screen-meta-links {
 }
 
 .wc-bridge-landing-page__title {
```

Comment:
How about introducing a more specific rule instead of using `!important` here? Such as the `.woocommerce.wc-bridge-landing-page .wc-bridge-landing-page__title`?

Reply:
Sounds good! Made changes in https://github.com/Automattic/wc-calypso-bridge/pull/1278/commits/9860d7ee171ac02a1f1a1f87e4c576b68092904a